.gitignore

```diff
@@ -73,4 +73,13 @@ lerna-debug.log
 
 #flatpak
 .flatpak-builder
-build-dir
\ No newline at end of file
+build-dir
+
+#Travis/IDE stuff
+.travis.yml
+.vs/
+
+#package-lock
+package-lock.json
```

Comment:
Please don't commit these (package-lock and package.json) as they need to be update if we install new packages.

Reply:
These lines have been taken out of .gitignore, should be tracked again.

Thanks!